guides/common/modules/proc_installing-the-dhcp-infoblox-module.adoc

```diff
@@ -29,9 +29,13 @@ You cannot use the `host` option without creating conflicts and, for example, be
 --foreman-proxy-plugin-dhcp-infoblox-record-type fixedaddress \
 --foreman-proxy-plugin-dhcp-infoblox-username admin
 ----
-. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}* and select the {SmartProxy} with the Infoblox DHCP module and click *Refresh*.
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *{SmartProxies}*, select the {SmartProxy} with the Infoblox DHCP module, and click *Refresh*.
 . Ensure that the *dhcp* features are listed.
 . For all domains managed through Infoblox, ensure that the DNS {SmartProxy} is set for that domain.
```

Comment:
Plural?
```suggestion
. For all domains managed through Infoblox, ensure that the DNS {SmartProxy} is set for those domains.
```

Though I wonder why DNS is suddenly here. The installer command above sets up DHCP, not DNS. So there's no guarantee that it actually is there. Perhaps drop it and leave it for the Infoblox DNS module?

Reply:
I was wondering about the same, I'll move it to the DNS module.